x-pack/plugins/security_solution/public/detections/components/rules/severity_mapping/translations.tsx

```diff
@@ -51,7 +51,8 @@ export const SEVERITY_DESCRIPTION = i18n.translate(
 export const SEVERITY_MAPPING_DESCRIPTION = i18n.translate(
   'xpack.securitySolution.alerts.severityMapping.mappingDescriptionLabel',
   {
-    defaultMessage: 'Map a value from the source event to a specific severity.',
+    defaultMessage:
+      'Override the default severity at left with a value derived from the source event.',
```

Comment:
Not sure we need `at left` here since we say `default severity` and that's the label of the field to the left. @benskelker thoughts here?

Same for `Risk Score` above.

Reply:
Also, the `with a value derived from the source event` isn't entirely accurate here, as we're not using that value from the source event for the `severity`, but rather checking the equality of that value with what the user provides and then use the corresponding severity level from that row (e.g. `low`/`medium`/`high`/`critical` )

So for the below, if `event.severity == 21` in the source event, we'll override the severity with `low`, not `21`:

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/87799756-d5c66a80-c80a-11ea-9245-cce607e5356d.png" />
</p>

Reply:
@spong  - yep, no need for `at left`.

Suggested text:

Use source event values to override the **Default severity**:

Use a source event value to override the **Default risk score**: